addons/nightvision/functions/fnc_refreshGoggleType.sqf

```diff
@@ -74,6 +85,8 @@ if (alive ACE_player) then {
             if (isNumber (_config >> QGVAR(bluRadius))) then {_blurRadius = getNumber (_config >> QGVAR(bluRadius));};
         };
         if (isNumber (_config >> QGVAR(generation))) then {_nvgGen = getNumber (_config >> QGVAR(generation));};
+        // Same as at line 72
```

Comment:
useless comment :D But...  as I look at that. Line 87 is also the same as 71.. Wait a second... That doesn't make any sense.
Oh no.. It does.. Ugh...
Maybe replace the confusing `exitWith` at line 68 with a `then` and a `else`?

Reply:
@dedmen i'm not really sure, 'cause i try'd to not touch original stuff where that's not necessary.
If it still understandable (meant exitWith), i guess that's not important for replacing.

Reply:
True. If you didn't do it then no need for you to fix it ^^

Reply:
@dedmen Well, want to say what `exitWith` more elegant way than `then` & `else`.